test/comparison-tests/customTransformer/webpack.config.js

```diff
@@ -0,0 +1,33 @@
+var path = require('path')
+
+// 1. import default from the plugin module
+var createStyledComponentsTransformer = require('./uppercaseStringLiteralTransformer').default;
+
+// 2. create a transformer;
+// the factory additionally accepts an options object which described below
+var styledComponentsTransformer = createStyledComponentsTransformer();
+
```

Comment:
The sample transformer exports the transformer itself, and not a factory function that `typescript-plugin-styled-components` does. So you don't need to call the function:
```ts
var uppercaseStringLiteralTransformer = require('./uppercaseStringLiteralTransformer').default;
```